.clang-format

```diff
@@ -18,6 +18,7 @@ ContinuationIndentWidth: 4
 IndentCaseLabels: true
 IndentWidth: 4
 IndentWrappedFunctionNames: false
+IndentExternBlock: NoIndent
```

Comment:
Note that this isn't actually being applied correctly right now because of a bug in clang-format that was fixed a few weeks ago. We need to wait for the next release to get the fix.
https://github.com/llvm/llvm-project/issues/36620
https://github.com/llvm/llvm-project/pull/75731